src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int buy = 0;
+        int supply = 0;
+        int result = 0;
+
+        try (BufferedReader reader =
+                     new BufferedReader(new FileReader(fromFileName))) { //try-with-resources
+
+            String line;
+
+            while ((line = reader.readLine()) != null) { // Читаємо файл, рядок за рядком
+
+                String[] parts = line.split("\\W+");
```

Comment:
The regex `\W+` used in `line.split("\\W+")` will split the line on any non-word character, which might not be the intended behavior if the file format uses a specific delimiter like a comma. Consider using a more specific delimiter, such as `","` if the file is CSV formatted.